modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -118,6 +118,51 @@ definition {
 		}
 	}
 
+	@description = "LPS-178192."
+	@priority = 5
+	test CanBeDisabledByInstanceSettings {
+		property portal.acceptance = "true";
+
+		task ("Given user with admin privileges") {
+			Navigator.openURL();
+		}
+
+		task ("And “Enable Accessibility Menu” is enabled") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "Instance Settings");
+
+			Click(
+				key_itemName = "Accessibility",
+				locator1 = "ListGroupItem#ITEM_TEXT");
+
+			AssertChecked.assertCheckedNotVisible(
+				checkboxName = "Enable Accessibility Menu",
+				locator1 = "Checkbox#ANY_CHECKBOX");
+		}
+
+		task ("When turn off Enable Accessibility Menu") {
+			Uncheck.uncheckToggleSwitch(
+				checkboxName = "Enable Accessibility Menu",
+				locator1 = "Checkbox#ANY_CHECKBOX");
+
+			Click(locator1 = "Button#UPDATE");
+
+			Alert.viewSuccessMessage();
+
+			Refresh();
+		}
+
+		task ("Then Accessibility Menu cannot be accessed") {
+			AccessibilityMenu.goToAccessibilityMenuQuickLink();
+
+			AssertElementNotPresent(
+				key_link = "Open Accessibility Menu",
+				locator1 = "AccessibilityMenu#QUICK_LINK");
+		}
```

Comment:
Still trying to find a better solution for it.

Reply:
Or is this enough? 

CC/ @john-co

Reply:
Good enough